src/PhenX.EntityFrameworkCore.BulkInsert/Metadata/PropertyAccessor.cs

```diff
@@ -1,66 +1,53 @@
-using System.Reflection.Emit;
+using System.Linq.Expressions;
 using System.Reflection;
 
 namespace PhenX.EntityFrameworkCore.BulkInsert.Metadata;
 
 internal static class PropertyAccessor
```

Comment:
I think this is a case where a micro-benchmark would be helpful, because it is difficult to get an understand from the general benchmarks.